barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_scalar.test.cpp

```diff
@@ -170,12 +169,23 @@ TYPED_TEST(CycleScalarTest, TestScalarFieldValidation)
 {
     using cycle_scalar = typename TestFixture::cycle_scalar;
     using ScalarField = typename TestFixture::ScalarField;
+    using field_t = typename TestFixture::field_t;
 
     TypeParam builder;
 
-    // Test with a valid scalar
+    // Manually create a valid scalar, bypassing the constructors that perform validation
     auto valid_scalar = ScalarField::random_element(&engine);
-    auto scalar = cycle_scalar::from_witness(&builder, valid_scalar);
+    uint256_t valid_scalar_u256 = uint256_t(valid_scalar);
```

Comment:
Changing this test to not use `cycle_scalar::from_witness` since that would now mean calling `validate_scalar_is_in_field` twice